packages/autoloader/src/AutoloadGenerator.php

```diff
@@ -325,7 +348,8 @@ private function getAutoloadPackageFile( $suffix ) {
  * Prepare all the classes for autoloading.
  */
 function enqueue_packages_$suffix() {
-	\$class_map = require_once dirname( __FILE__ ) . '/composer/autoload_classmap_package.php';
+	\$class_map    = require_once dirname( __FILE__ ) . '/composer/autoload_classmap_package.php';
+	\$package_meta = require_once dirname( __FILE__ ) . '/composer/autoload_meta.php';
```

Comment:
What I wanted to do here was to have a safe way for us to see which version of the package has generated this autoloader. Every autoloader from now on will have this file, which will tell it about its version. The other autoloaders we'll have to scan for, a simple `include` should be fine, because:
* other autoloaders may not have this file yet, and we ignore them
* the autoloaders that do have the file will let us know if their code should take precedence.

Reply:
We could also get the version of the `automattic/jetpack-autoloader` package from the `/composer/autoload_classmap_package.php` files. Is there an advantage to creating a new file with the `jetpack-autoloader` package version instead of using the information from the existing file?

Reply:
We could also get the version of the `automattic/jetpack-autoloader` package from the `/composer/autoload_classmap_package.php` files. Is there an advantage to creating a new file with the `jetpack-autoloader` package version instead of using the information from the existing file?

Reply:
This is now outdated, I'm going to resolve this thread.